daml-lf/spec/daml-lf-1.rst

```diff
@@ -386,12 +392,16 @@ We can now define all the literals that a program can handle::
 
 The literals represent actual DAML-LF values:
 
+* A ``LitNatType`` represents a natural number between ``0`` and
```

Comment:
Do we actually enforce these bounds anywhere?

Reply:
We will check during unserliazation.
See https://github.com/digital-asset/daml/pull/2298/files/33e6b1950b738efc874e833e075c7c7044405865#diff-55b02b80bc6355ad225115dd9046ce2eR302

We could relax this in the future if needed.